### PR TITLE
Implement `ctrlaltdel`

### PR DESCRIPTION
**Pull request by sisungo**

### CODE REVIEW

**Comment by sylvestre** [comment]
looks great, could you please add a few tests?

**Comment by sisungo** [comment]
Some unit tests were added.

File: tests/by-util/test_ctrlaltdel.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// This file is part of the uutils util-linux package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+// spell-checker:ignore (words) symdir somefakedir
+
+use crate::common::util::TestScenario;
+
+#[test]
+#[ignore]
```
[the remark this answers is not in the document]
Comment (sisungo) [reply tests/by-util/test_ctrlaltdel.rs]: Already changed. Now the test is no longer ignored and is only built on Linux.

**Comment by cakebaker** [comment]
Thanks for your PR :)

**Pull request merged**